packages/admin/resources/views/components/layouts/card.blade.php

```diff
@@ -1,13 +1,29 @@
 @props([
     'title' => null,
+    'width' => 'md'
 ])
 
 <x-filament::layouts.base :title="$title">
     <div @class([
         'flex items-center justify-center min-h-screen filament-login-page bg-gray-100 text-gray-900',
         'dark:bg-gray-900 dark:text-white' => config('filament.dark_mode'),
     ])>
-        <div class="w-screen max-w-md px-6 -mt-16 space-y-8 md:mt-0 md:px-2">
+        <div @class([
+            'w-screen px-6 -mt-16 space-y-8 md:mt-0 md:px-2',
+            match($width) {
+                'xs' => 'max-w-xs',
+                'sm' => 'max-w-sm',
+                'md' => 'max-w-md',
+                'lg' => 'max-w-lg',
+                'xl' => 'max-w-xl',
+                '2xl' => 'max-w-2xl',
+                '3xl' => 'max-w-3xl',
+                '4xl' => 'max-w-4xl',
+                '5xl' => 'max-w-5xl',
+                '6xl' => 'max-w-6xl',
+                '7xl' => 'max-w-7xl',
```

Comment:
We should add a `default => $width` here too since somebody could use a custom class and generate their own theme too.

Reply:
Done, thanks👍